client/app/nonComp/components/NonCompTabs.stories.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ReduxBase from 'app/components/ReduxBase';
+import { nonCompReducer, mapDataToInitialState } from '../reducers';
+
+import NonCompTabsUnconnected from './NonCompTabs';
+
+const ReduxDecorator = (Story) => {
+  const props = {
+    serverNonComp: {
+      businessLineUrl: 'vha',
+      baseTasksUrl: '/decision_reviews/vha',
```

Comment:
```suggestion
      baseTasksUrl: '/decision_reviews/vha',
      taskFilterDetails: {
        in_progress: {
          '["BoardGrantEffectuationTask", "Appeal"]': 1,
          '["DecisionReviewTask", "HigherLevelReview"]': 10,
          '["DecisionReviewTask", "SupplementalClaim"]': 3,
          '["VeteranRecordRequest", "Appeal"]': 1
        },
        completed: { }
      }
```
This will add in the filtering values.